spec/dummy/spec/rails_helper.rb

```diff
@@ -73,7 +73,7 @@
   # Capybara config
   #
   # selenium_firefox webdriver only works for Travis-CI builds.
-  default_driver = :selenium_chrome_headless
+  default_driver = :selenium_firefox
```

Comment:
@mirzamanib FYI: there's also a global DRIVER env in the .travis.yml that must be changed.